packs/data/feats.db/fortified-flesh.json

```
@@ -23,7 +23,20 @@
                 }
             ]
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "ActiveEffectLike",
+                "mode": "add",
+                "path": "flags.pf2e.livingMonolith.archetypeFeatCount",
+                "value": 1
+            },
+            {
+                "exceptions": "adamantine",
```

```suggestion
                "exceptions": ["adamantine"],
```
Exceptions is an array.